### PR TITLE
TSD-196 create GPA filter for secondary

### PR DESCRIPTION
**Pull request by esoliani**

### CODE REVIEW

File: app/scripts/controllers/dashboard.coffee [hunk 1]
```diff
@@ -117,6 +121,22 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', (
...
+        watchCompute 'gpaComputedMax',
+          dependencies: ['allSchools']
+          waitForPromise: true
+          computer: ([allSchools]) -> $q (resolve, reject) ->
+            MIN = 5
+            unless allSchools?
+              resolve MIN
+            else
+              allSchools.then ((schools) ->
+                ratios = schools
```
Comment (uniphil) [inline app/scripts/controllers/dashboard.coffee]: `ratios` should probably be `gpas`, or even just `values`
(also `maxRatio` below)

Comment (uniphil) [inline app/scripts/controllers/dashboard.coffee]: Alternatively, we could make a function that takes `(allSchoolsPromise, prop, defaultMax) ->` in `utils` and use it for both `ptratioComputedMax` and `gpaComputedMax` (and a few unit tests wouldn't hurt).

**Comment by uniphil** [comment]
Other than the copied `ratios` names, this looks great!

File: app/scripts/controllers/dashboard.coffee (hunk 1 above)
Comment (esoliani) [inline app/scripts/controllers/dashboard.coffee]: This is the best option. I consider it but I was lasy

**Comment by uniphil** [comment]
:ok_hand:

**Pull request merged**